eng/Versions.props

```diff
@@ -125,7 +125,7 @@
     <SystemSecurityPrincipalWindowsVersion>5.0.0</SystemSecurityPrincipalWindowsVersion>
     <SystemServiceModelPrimitivesVersion>4.9.0-rc2.21473.1</SystemServiceModelPrimitivesVersion>
     <SystemTextJsonVersion>7.0.0-preview.2.22103.2</SystemTextJsonVersion>
-    <SystemRuntimeCompilerServicesUnsafeVersion>7.0.0-preview.2.22103.2</SystemRuntimeCompilerServicesUnsafeVersion>
+    <SystemRuntimeCompilerServicesUnsafeVersion>6.0.0</SystemRuntimeCompilerServicesUnsafeVersion>
```

Comment:
please also remove the entry for `System.Runtime.CompilerServices.Unsafe` from Version.Details.xml, otherwise the bot will try to auto-update the version here.

Reply:
Done.